runtime/compiler/il/J9Node.hpp

```diff
@@ -381,12 +381,14 @@ class OMR_EXTENSIBLE Node : public OMR::NodeConnector
 
 protected:
 
-   /// \note update the following routines when adding field to DecimalInfo
-   ///       isDecimalSizeAndShapeEquivalent()
-   ///       isSignStateEquivalent()
-   ///       resetDecimalSignFlags()
-   ///       transferSignState()
-   /// Also be sure to initialize any new field in the Node.cpp constructor
+  /**
+   * \note update the following routines when adding field to DecimalInfo
+   *       isDecimalSizeAndShapeEquivalent()
+   *       isSignStateEquivalent()
+   *       resetDecimalSignFlags()
+   *       transferSignState()
+   * Also be sure to initialize any new field in the Node.cpp constructor
+   */
```

Comment:
Please indent these lines one more space so that the `/` on the first line lines up with the word `struct`